project.template.yaml

```
@@ -93,7 +93,7 @@ network:
         types:
           [
             {
-              minmax: [3001, 3001],
+              minmax: [3001, 3002],
```

```suggestion
              minmax: [3002, 3002],
```
Then there should be `[3000,3001] ` below